src/cmd/mod.rs

```diff
@@ -20,57 +23,58 @@ pub enum SelectResult {
     Selected(String),
 }
 
-pub fn select_or_list_context(installed: &mut Installed) -> Result<SelectResult> {
+pub fn select_context(skim_options: &SkimOptions, installed: &mut Installed) -> Result<SelectResult> {
```

Comment:
The function was called `select_or_list_context` because it just lists the items when the output is not a terminal. Can you restore that?